collections/_article/dev-snapshot-godot-4-4-beta-3.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Dev snapshot: Godot 4.4 beta 3"
+excerpt: Picking up the pace!
+categories: [pre-release]
+author: Thaddeus Crews
+image: /storage/blog/covers/dev-snapshot-godot-4-4-beta-3.webp
+image_caption_title: Ballionaire
+image_caption_description: A game by newobject
+date: 2025-02-06 12:00:00
+---
+
+Woah, another beta build already? You're not imagining things, it's only been a week since our last release. The team is fully in the swing of getting release-blockers merged, so we're able to expedite our output! What's more, our community has been on the ball with submitting regression reports; thanks to everyone who's involved themselves!
+
+While progress has been at a rate we're all excited about, it's not over yet! If you haven't already, we encourage users who haven't engaged with the beta releases to do so and help us catch the last few stragglers. It won't be long now until we can start bringing out the release candidates!
+
```

Comment:
That really is an awful lot of exclamation marks!

Reply:
The narrator has been successfully sedated